src/main/pages/che-7/administration-guide/assembly_using-organizations.adoc

```diff
@@ -0,0 +1,224 @@
+
+:context: using-organizations
+
+[id="organizations-in-eclipse-che"]
+== Organizations in Eclipse Che
+
+Organizations allow administrators to group Eclipse Che users and allocate resources. The system administrator controls and allocates resources and permissions within the administrator dashboard.
+
+
+[id="roles-in-an-organization"]
+=== Roles in an organization
+
+A user can have the following roles in an organization:
+
+Members:: Create workspaces, manage their own workspaces, and use any workspaces they have permissions for.
+Administrators:: Manage the organization, members, resources, and sub-organization, and can edit settings.
+System Administrators:: Create root organizations, manages resources, members and sub-organizations. System administrators have more permissions than the administrators and members. 
+
+
+[id="root-organizations-and-sub-organizations"]
+=== Root organizations and sub-organizations 
+
+The top-level organizations are called root organizations.   Multiple root organizations can be created.  Any organization can have zero to a set number of sub-organizations.  Only the system administrator can create root organizations and manage the resources of the root organization. 
+
+
+[id="creating-an-organization"]
+=== Creating an organization
+
+Only the system administrator can create root organizations. An administrator can create sub-organizations. 
+
+To create an organization:
+
+.  Click the menu in the left sidebar.  A new page displays all the organizations in your system.
+
+.  Click on the upper-left button to create a new organization.
+
+
+[id="displaying-the-list-of-organizations"]
+=== Displaying the list of organizations
+
+The *Organization* page displays a list of all the organizations. The list contains the following information for each organization: number of members, total RAM, available RAM, and number of sub-organizations.
+
+
+[id="adding-members-to-organizations"]
+=== Adding members to organizations
+
+To add members to an organization:
+
+. Click the *Add* button to add a member.  A new pop-up window displays.  You can change the role of a member or remove them from the organization at any time. 
+
+. Enter the new member name.  
+
+[NOTE]
+====
+Users with the green checkmark beside their name already have an Eclipse Che account and can be added to the organization. Users without a checkmark do not have an account and cannot be added into the organization.
+====
+
+
+[id="workspaces-in-organizations"]
+=== Workspaces in organizations
+
+A workspace is created inside of an organization and uses the resources of the organization. The workspace creator chooses the organization on the *Workspace Creation* page.  
+
+
+[id="setting-email-notifications"]
+=== Setting email notifications
+
+To send notifications from the Che server when a user joins or leaves an organization, you can do either of the following:
+
+* Configure the SMTP server in the `che.env` file.  
```

Comment:
this is not up-to-date afaik. cc @skabashnyuk

Reply:
I don't see any mention of mail related settings in the properties files listed in  https://github.com/eclipse/che/issues/15356. So I assume the complete "Setting email notifications" should stay in the attic.